platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/reconnect/VirtualMapReconnectTestBase.java

```diff
@@ -247,13 +247,13 @@ public BreakableDataSource(final BrokenBuilder builder, final VirtualDataSource<
 
         @Override
         public void saveRecords(
-                final long firstLeafPath,
-                final long lastLeafPath,
-                final Stream<VirtualHashRecord> pathHashRecordsToUpdate,
-                final Stream<VirtualLeafRecord<TestKey, TestValue>> leafRecordsToAddOrUpdate,
-                final Stream<VirtualLeafRecord<TestKey, TestValue>> leafRecordsToDelete)
+                long firstLeafPath,
```

Comment:
Is there a reason that you removed the `final`s on these parameters?

Reply:
That's a refactoring artifact. Thank you for pointing it out.